server/command/command.go

```diff
@@ -1285,20 +1598,35 @@ func (r *Runner) generateTestData(numActiveIncidents, numEndedIncidents int, beg
 		return
 	}
 
+	var playbooks []playbook.Playbook
 	if len(playbooksResult.Items) == 0 {
-		r.postCommandResponse("You are not a member of any playbook. Create at least one playbook before generating the test data.")
-		return
-	}
+		dummyPlaybook := dummyListPlaybooks[rand.Intn(len(dummyListPlaybooks))]
+		dummyPlaybook.TeamID = r.args.TeamId
+		newPlayBookID, err := r.playbookService.Create(dummyPlaybook, r.args.UserId)
```

Comment:
Nit: we usually write "book" in lowercase
```suggestion
		newPlaybookID, err := r.playbookService.Create(dummyPlaybook, r.args.UserId)
```